utils/training_utils.py

```diff
@@ -0,0 +1,95 @@
+"""
+This utility file contains the function to load in training data as final or shuffled to be used in training a machine learning model.
+"""
+
+import numpy as np
+import pandas as pd
+import pathlib
+from typing import Tuple
+
+
+# set numpy seed to make random operations (shuffling data) reproducible
+np.random.seed(0)
+
+
+def get_X_y_data(
+    df: pd.DataFrame, label: str, shuffle: bool = False
+) -> Tuple[np.array, np.array]:
+    """Get X (feature space) and labels (predicting class) from pandas Data frame
+
+    Args:
+        df (pd.DataFrame): Data frame containing morphology.
+        label (str): Name of the Metadata column being used as the predicting class
+        shuffle (bool, optional): Shuffle the feature columns to get a shuffled dataset. Defaults to False.
+
+    Returns:
+        Tuple[np.array, np.array]: Returns np.arrays for the feature space (X) and the predicting class (y)
+    """
+    # Remove "Metadata" columns from df, leaving only the feature space
+    X = df.loc[:, ~df.columns.str.contains("Metadata")].values
+
+    # Extract class label
+    y = df.loc[:, [label]].values
+    # Make labels as array for use in machine learning
+    y = np.ravel(y)
+
+    # If shuffle is True, shuffled the columns independently for the feature space for training a model
+    if shuffle == True:
+        for column in X.T:
+            np.random.shuffle(column)
+
+    return X, y
```

Comment:
This is inplace for the shuffling, ensure that the non-shuffled model is run first to avoid shuffle overwriting

Reply:
Great idea! I have added `copy()` to avoid this issue! Thanks!